src/Sarif.Viewer.VisualStudio.Interop/SarifViewerInterop.cs

```diff
@@ -111,7 +111,7 @@ public Task<bool> OpenSarifLogAsync(string path)
 
             return this.CallServiceApiAsync(ViewerLoadServiceInterfaceName, (service) =>
             {
-                service.LoadSarifLogs(path);
```

Comment:
>LoadSarifLogs [](start = 24, length = 13)

This was a misspelling, missed because `service` is `dynamic` and this method was (apparently) not tested. There is no overload of `ILoadSarifLogService.LoadSarifLogs` that takes a single `string` argument. This call caused a runtime error.

Reply:
Not entirely clear here, is there a test gap? If so, where is it?

---
In reply to: [514514633](https://github.com/microsoft/sarif-visualstudio-extension/pull/234#discussion_r514514633) [](ancestors = 514514633)

Reply:
A test gap. There are no tests at all of the interop assembly (they'd have to be integration tests because it expects the viewer at the other end).

---
In reply to: [514524509](https://github.com/microsoft/sarif-visualstudio-extension/pull/234#discussion_r514524509) [](ancestors = 514524509,514514633)